test/cpp/jit/test_gpu.cpp

```diff
@@ -12243,7 +12243,7 @@ TEST_F(NVFuserTest, FusionWelfordOp_CUDA) {
   outputs[1] /= N;
 
   testValidate(
-      &fusion,
+      fe.kernel(),
```

Comment:
Instead of changing this, can we change `testValidate` to take the kernel pointer from the passed fusion pointer? That way, we could always pass `&fusion`.

Reply:
We could probably just read it as an int64_t for the purposes of testValidate, but we can't just send fusion in, unless we send the copy fe has because that's the one that's actually a kernel.

Reply:
Ah, right, never mind, I misunderstood the logic.